recipes/spherely/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "spherely" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/benbovy/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 84d0634b6e22bfffc4d6730c70a7f4bfaa7a1ca05e57aa6fd83cd71421247b23
+
+build:
+  number: 0
+  skip: true  # [py<310 or python_impl == 'pypy']
+  script: python -m pip install . -vv
+
+requirements:
+  build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - numpy                                  # [build_platform != target_platform]
+    - {{ compiler('cxx') }}
+    - {{ stdlib('c') }}
+    - cmake
+    - make  # [linux]
+  host:
+    - python
+    - pip
+    - scikit-build-core
+    - pybind11 >=2.10.0
+    - s2geometry >=0.11.1
+    - s2geography >=0.1.2
+    - numpy
+  run:
+    - python
+    - {{ pin_compatible('s2geometry', max_pin='x.x') }}
+    - {{ pin_compatible('s2geography', max_pin='x.x') }}
```

Comment:
```suggestion
```

Reply:
Hmm shouldn't there still be those run dependencies?

```yaml
  - s2geometry
  - s2geography
```

CI is failing..

Reply:
Once they have run_exports and the respective builds have hit the CDN, then they should be automatically as run dependencies.

Reply:
OK thanks. I opened https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/6379. Not sure if that is the right way to do it, though.

> Once they have run_exports and the respective builds have hit the CDN, then they should be automatically as run dependencies.

How does this work in staged recipes? Do we need to re-run the CI manually at some point in this PR?